.gitignore

```diff
@@ -23,3 +23,4 @@
 /.pydevproject
 /.idea
 /*.iml
+*.swp
```

Comment:
Ah, probably best to exclude it here, and add it to all the .gitignore files simultaneously in a separate commit. Just because it's weird to add it to one, but not the others.